src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/TalismanListener.java

```diff
@@ -149,7 +149,7 @@ public void onItemBreak(PlayerItemBreakEvent e) {
             // Did the tool in our hand broke or was it an Armorpiece?
             if (!inv.getItem(inv.getHeldItemSlot()).equals(e.getBrokenItem())) {
                 for (int s : armorSlots) {
-                    if (inv.getItem(s).equals(e.getBrokenItem())) {
+                    if (e.getBrokenItem().equalst(inv.getItem(s))) {
```

Comment:
That's a typo right there... "equalst".
You should always test your changes before commiting them. Especially seeing if it even compiles properly.